tests/UI/accents.ts

```diff
@@ -0,0 +1,198 @@
+        }
+
+        tearDown() {
+            document.body.parentElement.classList.remove("win-ui-light");
+            testElement && testElement.parentElement && testElement.parentElement.removeChild(testElement);
+            testElement = null;
+            Accents._reset();
+        }
+
+        testCreateAccentRuleSimple(complete) {
+            LiveUnit.Assert.areNotEqual(Accents._colors[Accents.ColorTypes.accent], getComputedStyle(testElement).color);
+
+            Accents.createAccentRule("#accent-test-element", [{ name: "color", value: Accents.ColorTypes.accent }]);
+            WinJS.Promise.timeout().done(() => {
+                LiveUnit.Assert.isTrue(getDynamicStyleElement().textContent.indexOf("#accent-test-element") !== -1);
+                LiveUnit.Assert.areEqual(Accents._colors[Accents.ColorTypes.accent], getComputedStyle(testElement).color);
+                complete();
+            });
+        }
+
+        testCreateAccentRuleInverse(complete) {
+            document.body.parentElement.classList.add("win-ui-light");
+
+            var cs = getComputedStyle(testElement);
+            LiveUnit.Assert.areNotEqual(Accents._colors[Accents.ColorTypes.listSelectRestInverse], cs.backgroundColor);
+            LiveUnit.Assert.areNotEqual(Accents._colors[Accents.ColorTypes.listSelectHoverInverse], cs.columnRuleColor);
+            LiveUnit.Assert.areNotEqual(Accents._colors[Accents.ColorTypes.listSelectPressInverse], cs.outlineColor);
+
+            Accents.createAccentRule("#accent-test-element", [
+                { name: "background-color", value: Accents.ColorTypes.listSelectRest },
+                { name: "column-rule-color", value: Accents.ColorTypes.listSelectHover },
+                { name: "outline-color", value: Accents.ColorTypes.listSelectPress },
+            ]);
+
+            Accents.createAccentRule("#accent-test-element", [{ name: "color", value: Accents.ColorTypes.listSelectHover }]);
+            WinJS.Promise.timeout().done(() => {
+                LiveUnit.Assert.areEqual(Accents._colors[Accents.ColorTypes.listSelectRestInverse], getComputedStyle(testElement).backgroundColor);
+                LiveUnit.Assert.areEqual(Accents._colors[Accents.ColorTypes.listSelectHoverInverse], getComputedStyle(testElement).columnRuleColor);
+                LiveUnit.Assert.areEqual(Accents._colors[Accents.ColorTypes.listSelectPressInverse], getComputedStyle(testElement).outlineColor);
+                complete();
+            });
+        }
+
+        testCreateAccentRuleInverseNoLeadingToken(complete) {
+            LiveUnit.Assert.areNotEqual(Accents._colors[Accents.ColorTypes.listSelectHover], getComputedStyle(testElement).color);
+
+            Accents.createAccentRule("randomTag", [{ name: "color", value: Accents.ColorTypes.listSelectHover }]);
+            WinJS.Promise.timeout().done(() => {
+                LiveUnit.Assert.isTrue(getDynamicStyleElement().textContent.indexOf("randomTag") !== -1);
```

Comment:
isTrue(x !== -1)
isFalse(x !== -1)

It's hard to tell which is which at a glance. And this is a common pattern. Perhaps you should have a couple of helpers:
  - assertContains
  - assertDoesntContain